tests/network/provider_migration/libprovider.py

```
@@ -31,18 +40,31 @@ def __exit__(self, exc_type, exc_val, exc_tb):
     def content(self) -> vim.ServiceInstanceContent:
         return self._client.RetrieveContent()
 
-    def power_on_vm(self, vm_name: str) -> None:
-        vm = self._get_vm_by_name(name=vm_name)
+    def power_on_vm(self, vm_name: str) -> vim.VirtualMachine:
+        LOGGER.info(f"Powering on VM '{vm_name}'")
```

Ruff (0.14.10)

Logging statement uses f-string

(G004)